src/script.ts

```diff
@@ -164,15 +164,17 @@ function updateTheme(): void {
 // Called every second
 function updateClock(): void {
     const time = new Date();
-    const clockTextElm = document.getElementById("clock-text");
+    const clockTextElms = document.getElementsByClassName("clock-text");
     let formattedTimeText = time.toLocaleString("en-US", { hour: "numeric", minute: "numeric", hour12: true });
     if (time.getSeconds() % 2) { // to make the blinking effect
         formattedTimeText = formattedTimeText.replace(":", " ");
     }
-    if (clockTextElm == null) {
+    if (clockTextElms == null) {
```

Comment:
This will never be null, so the if statement isn't necessary. Just loop through the objects in the collection.

Reply:
fixed